[feat] Add Audit trail documentation (#775)
grafana/k6-docs: src/data/markdown/docs/03 cloud/03 Manage/06 Audit Trail.md

---
title: 'Audit Trail'
excerpt: 'Monitor actions performed by your organization'
slug: '/cloud/manage/audit-trail'
---

> The *Audit Trail* can provide valuable insights into the activity taking place within your organization.

Reviewer: ```suggestion
The *Audit Trail* can provide valuable insights into the activity within your organization.
```

> ⭐️ Audit Trail is available on annual Pro and Enterprise plans.

> To view the Audit Trail, go to  **Manage > Audit Trail** section.

Reviewer: ```suggestion
To view the Audit Trail, go to  **Manage > Audit Trail**.
```

> ## What is an Audit trail

Reviewer 2: ```suggestion
## What is an Audit trail?
```

Reviewer: 5 weeks ago, I would've agreed. But in that time, I've started to be wary of putting questions in sub-headings, because people scan pages looking at the left side of the column. So, placing "What is" in the column makes the content a little more difficult to scan.

Not my idea, by the way. I stole it from here: https://content-guide.18f.gov/our-approach/structure-the-content/
And it made sense to me, since people [read online in F-shaped patterns](https://www.nngroup.com/articles/f-shaped-pattern-reading-web-content/).

Does it make sense?

Reviewer 2: After your changes, it makes more sense.

> ## What is an Audit trail

Reviewer: ```suggestion
## A list of system events and activities

```

Reviewer: Maybe instead of list, we should use "Log" here? I'm thinking that some people may conceive of audit trails as logging, so adding this word somewhere may help searchers find the page.

If this is an inaccurate use of use of the word "log," of course, discard this suggestion.

> An audit trail provides a chronological list of system events and user activities. This is a great way
> for organization owners to keep an eye on operations being performed by their users and
> to help trace specific events.

Reviewer: ```suggestion
An audit trail provides a chronological list of system events and user activities. 
Organization owners can use audit trails:
- To keep an eye on the operations that their users perform
- To help trace specific events.
```

## Reconstructing Events

> By reading through the audit trail you can start to piece together how a series 

Reviewer: ```suggestion
By reading through the audit trail, you can piece together how a series 
```

> of events perhaps led to a test being deleted, a member losing access to a project, subscription 
> or billing settings being altered, and much more.

Reviewer: ```suggestion
of events perhaps led to how a test was deleted, a member lost access to a project, subscription 
or billing settings were altered, and much more.
```

> ## Threat Detection
> Perhaps you've noticed suspicious activity and you want to rule out the possibility 
> something or someone has gained unauthorized access to your organization. You can scan 
> the audit trail for abnormalities and even check the IP addresses correlate to what you have 
> registered for your users.

Reviewer: ```suggestion
## Threat Detection

If you've noticed suspicious activity,
you can use audit logs to rule out the possibility that something or someone gained unauthorized access to your organization.
You can scan the audit trail for abnormalities and even check whether the IP addresses correlate to what you have 
registered for your users.
```

---
title: 'Audit Trail'
excerpt: 'Monitor actions performed by your organization'
slug: '/cloud/manage/audit-trail'
---

The *Audit Trail* can provide valuable insights into the activity within your organization.

> ⭐️ Audit Trail is available on annual Pro and Enterprise plans.

To view the Audit Trail, go to  **Manage > Audit Trail**.

## A log of system events and activities

An audit trail provides a chronological list of system events and user activities. 

> Organization owners can use audit trails:
> - To keep an eye on the operations that their users perform
> - To help trace specific events.

Reviewer: ```suggestion
Organization owners can use audit trails to keep an eye on the operations that their users perform and to trace events.
```

## Reconstructing Events

> By reading through the audit trail, you can piece together how a series   
> of events perhaps led to how a test was deleted, a member lost access to a project, subscription 
> or billing settings were altered, and much more.

Reviewer: ```suggestion
With the audit trail, you can piece together how a series of events perhaps led to an event.
Use cases include investigations into the following:
- How a test was deleted
- How a member lost access to a project,
- How a subscription or billing settings were altered.
```
